api/api_test.go

```diff
@@ -240,6 +287,44 @@ func TestCreateDeleteNetwork(t *testing.T) {
 	if errRsp != &successResponse {
 		t.Fatalf("Unexepected failure: %v", errRsp)
 	}
+
+	// Create with labels
+	labels := []string{
+		netlabel.EnableIPv6, "true",
+		"io.docker.network.bridgename", "abc",
+		"io.docker.network.bridge.io.docker.network.bridge.allow_non_default_bridge", "true",
```

Comment:
Reading quickly over the code, but these are meant as "failing" tests?  If not; these labels don't match the names defined earlier :smile: